bin/docker-push

```diff
@@ -17,7 +17,7 @@ then
   exit 1
 fi
 
-docker tag $DOCKERTAG:latest ${EXTERNAL_REGISTRY_BASE_DOMAIN}/${REPOSITORY_NAME}:$CI_SHA1
+docker tag -f $DOCKERTAG:latest ${EXTERNAL_REGISTRY_BASE_DOMAIN}/${REPOSITORY_NAME}:$CI_SHA1
```

Comment:
So after some research, I don't think we should be using `tag -f`.  This was deprecated in Docker 1.10 and removed in 1.12.

https://docs.docker.com/engine/deprecated/

> To make tagging consistent across the various docker commands, the -f flag on the docker tag command is deprecated. It is not longer necessary to specify -f to move a tag from one image to another. Nor will docker generate an error if the -f flag is missing and the specified tag is already in use.

Could you remove this from your PR?